server/server.go

```diff
@@ -896,7 +896,7 @@ func (s *Server) Open() error {
 	}
 
 	if err := s.LoadService.Load(); err != nil {
-		return fmt.Errorf("loading tasks/templates/handlers: %s", err)
```

Comment:
I think its fine for the server to refuse to start if it has bad configuration, but we do not want it to crash from a simple typo.

Reply:
not sure exactly what to do with this. Should I keep the error here and do some checking of the type of error, or just leave it as is?

Reply:
What types of errors would you check for? Seems like if it errors at all it should return the error.